<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicPlayDlg</class>
 <widget class="QDialog" name="MusicPlayDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音乐播放器2021</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/Icon/1270514.ico</normaloff>res/Icon/1270514.ico</iconset>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonpausestart">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonstop">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>正在播放...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonbrowser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件夹</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="musicprogress">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>631</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="columnSlider">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>410</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>410</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>音量：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPre">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxMusicList">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>631</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>播放列表</string>
   </property>
   <widget class="QListView" name="musicListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>611</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>速度：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>100%</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>125&amp;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>150%</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
